app/components/UserValidator/index.js

```diff
@@ -56,39 +60,73 @@ class Validator extends Component<Props, State> {
         }
       )
       .then(() => {
-        if (this.input) {
-          this.input.focus();
+        if (input) {
+          input.focus();
         }
       });
   };
 
-  render() {
-    return (
-      <div>
-        <div
-          className={cx(styles.overlay, {
-            [styles.shown]: this.state.showCompleted,
-          })}
-        >
-          <h3>
-            Tusen takk! Kos deg{' '}
-            <span role="img" aria-label="smile">
-              😀
-            </span>
-          </h3>
-          <i className="fa fa-check" />
-        </div>
-        <SearchPage
-          {...this.props}
-          placeholder="Skriv inn brukernavn eller navn"
-          handleSelect={this.handleSelect}
-          inputRef={(input) => {
-            this.input = input;
+  return (
+    <div>
+      <div
+        className={cx(styles.overlay, {
+          [styles.shown]: completed,
+        })}
+      >
+        <h3>
+          Tusen takk! Kos deg{' '}
+          <span role="img" aria-label="smile">
+            😀
+          </span>
+        </h3>
+        <i className="fa fa-check" />
+      </div>
+      <Modal
+        contentClassName={styles.scannerModal}
+        show={showScanner}
+        onHide={() => setShowScanner(false)}
+      >
+        <h1>Scan ABA-ID</h1>
+        <QrReader
+          onResult={(res, error) => {
+            if (!processing && res) {
+              handleSelect({
+                username: res.getText(),
+                result: '',
+                color: '',
+                content: '',
+                icon: '',
+                label: '',
+                link: '',
+                path: '',
+                picture: '',
+                value: '',
+              });
+            }
+            if (error) {
+              console.info(error);
+            }
           }}
+          constraints={{ facingMode: 'environment' }}
         />
-      </div>
-    );
-  }
-}
+      </Modal>
+      <Button
+        className={styles.scannerButton}
+        onClick={() => setShowScanner(true)}
+      >
+        <Icon name="qr-code" size={18} />
+        Vis Scanner
```

Comment:
I prefer to not capitalize every word.
```suggestion
        Vis scanner
```

Reply:
✅